src/kOS.Safe/Encapsulation/ListValue.cs

```diff
@@ -95,6 +95,7 @@ private void ListInitializeSuffixes()
             AddSuffix("INSERT",   new TwoArgsSuffix<ScalarValue, T>     ((index, toAdd) => Collection.Insert(index, toAdd)));
             AddSuffix("REMOVE",   new OneArgsSuffix<ScalarValue>        (toRemove => Collection.RemoveAt(toRemove)));
             AddSuffix("SUBLIST",  new TwoArgsSuffix<ListValue, ScalarValue, ScalarValue>(SubListMethod));
+            AddSuffix("JOIN",     new OneArgsSuffix<StringValue, StringValue>     ((separator) => Join(separator)));
```

Comment:
If you make the `Join` method accept and return return a `StringValue` instead of a `string`, you can remove the anonymous method here and simply pass it as 
```
AddSuffix("JOIN", new OneArgsSuffix<StringValue, StringValue>(Join));
```